recipe/data.go

```diff
@@ -53,3 +53,24 @@ var CPIH = Response{
 	},
 }
 
+
+var MIDYEARPOPEST = Response{
+	ID:     "40AA070E-7A43-4EC5-A1FC-84CEA2BC4461",
+	Alias:  "MIDYEARPOPEST",
+	Format: "v4",
+	InputFiles: []file{
+		{"MIDYEARPOPEST v4"},
+	},
+	OutputInstances: []instance{
+		{
+			DatasetID: "midyearpopest",
+			Editions:  []string{"time-series"},
+			Title:     "Mid Year Population Estimates",
+			CodeLists: []CodeList{{ID: "time", Name: "time", HRef: "http://localhost:22400/code-lists/time"},
+				{ID: "midyearpopgeography", Name: "geography", HRef: "http://localhost:22400/code-lists/midyearpop"},
```

Comment:
For code list and dataset ids it would be good to use hyphens to delineate - `mid-year-pop-geography` so we can clearly see the dimension name